Helper_Scripts/Samples/Grafana/README.md

```
@@ -14,6 +14,7 @@ Dashboards to load (copy into `/var/lib/grafana/dashboards` in your Grafana cont
 - `Docs/Deployment/Monitoring/mcp-dashboard.json`
 - `Docs/Deployment/Monitoring/web-scraping-dashboard.json`
 - `Docs/Deployment/Monitoring/streaming-dashboard.json`
+ - `Docs/Deployment/Monitoring/chatbook-tools-dashboard.json`
```

markdownlint-cli2 (0.18.1)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)